src/AllReady/AllReady.csproj

```diff
@@ -31,7 +31,7 @@
     <RazorViewsFiles Include="..\..\submodules\AllReady\AllReadyApp\Web-App\AllReady\**\*.cshtml" />
   </ItemGroup>
 
-  <Target Name="CopyFiles" BeforeTargets="PreBuildEvent">
+  <Target Name="CopyFiles" BeforeTargets="Restore">
```

Comment:
You could make this `BeforeTargets="PrepareForPublish"`. The inputs to Razor compilation aren't calculated until after `PrepareForPublish`

Reply:
@pranavkm but what is first? PreBuildEvent or PrepareForPublish? I ended up using Restore because when I was using PreBuildEvent the first publish was not containing the razor views. So I would have to build it twice to get what I want.

Reply:
I'm not entirely sure where `PreBuildEvent` is located. I was going by the target the precompilation tool wires up to.